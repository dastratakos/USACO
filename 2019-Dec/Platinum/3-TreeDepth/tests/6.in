19 85 794215811
