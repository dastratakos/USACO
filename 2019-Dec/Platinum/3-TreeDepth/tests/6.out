511056080 637508276 190300150 29210812 510749361 355748507 413475756 620017804 752917392 404850167 616149292 578505796 178005322 617303400 202137717 65000981 76969655 84890253 330316897
